<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
    <line>
      <![CDATA[开始准备测试数据 ]]>
    </line>
    <line>
      <![CDATA[资料收取 12_39]]>
    </line>
    <line>
      <![CDATA[<br/><img src="D:\work_space\js_de\debug\SeleniumConfig\bin\Debug\SootScreens\2019-03-21\12_39\资料收取 12_39.png"  hight="900" width="900" />]]>
    </line>
  </reporter-output>
  <suite name="断言调试-房地登记-首次登记" duration-ms="104" started-at="2019-03-21T04:40:28Z" finished-at="2019-03-21T04:40:28Z">
    <groups>
    </groups>
    <test name="断言详情查看" duration-ms="104" started-at="2019-03-21T04:40:28Z" finished-at="2019-03-21T04:40:28Z">
      <class name="Test.DataProviderSet_Assert">
        <test-method status="PASS" signature="insetToExcel()[pri:0, instance:Test.DataProviderSet_Assert@41beb473]" name="insetToExcel" is-config="true" duration-ms="426" started-at="2019-03-21T12:40:28Z" finished-at="2019-03-21T12:40:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- insetToExcel -->
        <test-method status="FAIL" signature="star(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Test.DataProviderSet_Assert@41beb473]" name="star" duration-ms="28" started-at="2019-03-21T12:40:28Z" data-provider="AssertData" finished-at="2019-03-21T12:40:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[申请人信息]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[申请人]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[断言申请人信息]]>
              </value>
            </param>
          </params>
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<"申请人[]"> but was:<"申请人[信息]">]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<"申请人[]"> but was:<"申请人[信息]">
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at Test.DataProviderSet_Assert.star(DataProviderSet_Assert.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at configSelenium.ConfigSeleniumStart.dotestng(ConfigSeleniumStart.java:1042)
	at configSelenium.ConfigSeleniumStart.main(ConfigSeleniumStart.java:183)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- star -->
        <test-method status="FAIL" signature="star(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Test.DataProviderSet_Assert@41beb473]" name="star" duration-ms="0" started-at="2019-03-21T12:40:28Z" data-provider="AssertData" finished-at="2019-03-21T12:40:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[不动产单元信息]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[不动产单元]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[不动产单元信息断言]]>
              </value>
            </param>
          </params>
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<"不动产单元[]"> but was:<"不动产单元[信息]">]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<"不动产单元[]"> but was:<"不动产单元[信息]">
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at Test.DataProviderSet_Assert.star(DataProviderSet_Assert.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at configSelenium.ConfigSeleniumStart.dotestng(ConfigSeleniumStart.java:1042)
	at configSelenium.ConfigSeleniumStart.main(ConfigSeleniumStart.java:183)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- star -->
      </class> <!-- Test.DataProviderSet_Assert -->
    </test> <!-- 断言详情查看 -->
  </suite> <!-- 断言调试-房地登记-首次登记 -->
</testng-results>
